app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -2583,13 +2589,15 @@ object TelemetryWrapper {
             value = "",
             extras = [
                 TelemetryExtra(name = Extra.LINK, value = "${Extra_Value.URL},${Extra_Value.DEEPLINK},null"),
-                TelemetryExtra(name = Extra.MESSAGE_ID, value = "message_id")
+                TelemetryExtra(name = Extra.MESSAGE_ID, value = "message_id"),
+                TelemetryExtra(name = Extra.BACKGROUND, value = "true|false")
```

Comment:
As @cnevinc mentioned, we should use `,` to separate value instead of `|` in the document.